source/renderer/app/components/hardware-wallet/HardwareWalletStatus.js

```diff
@@ -196,6 +201,10 @@ export default class HardwareWalletStatus extends Component<Props, State> {
       hwDeviceStatus === HwDeviceStatuses.UNSUPPORTED_DEVICE ||
       hwDeviceStatus === HwDeviceStatuses.VERIFYING_TRANSACTION_FAILED;
 
+    const hasPassphraseLabel =
+      hwDeviceStatus === HwDeviceStatuses.EXPORTING_PUBLIC_KEY ||
```

Comment:
@tomislavhoracek maybe in a different task, we could move `WalletDiscovery`, `WalletSyncStateStatuses`, `WalletDelegationStatuses` and  `HwDeviceStatus` into config files

Reply:
@daniloprates hmm interesting idea. Maybe we can sync about that and see together what could be the best place to isolate?